### PR TITLE
Remove ConsentiumThingsDalton Library

### PR DESCRIPTION
**Pull request by debjyotiC**
Please remove the library ConsentiumThingsDalton from the Arduino Library Index. Repository URL: https://github.com/ConsentiumInc/ConsentiumThingsDalton The `library.properties` file has been updated to mark the library as obsolete.

Thank you.

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @debjyotiC. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by umbynos — APPROVED** [review]
Done in https://github.com/arduino/library-registry/pull/4857#issuecomment-2255876008

**Pull request merged**
